images/dind/Dockerfile

```diff
@@ -57,7 +57,9 @@ RUN dnf -y update && dnf -y install\
  glibc-langpack-en\
  iptables\
  openssh-clients\
- openssh-server
+ openssh-server\
+ net-tools\
```

Comment:
net-tools is all stuff that has been deprecated by newer utilities (mostly in iproute2)

Also, we don't run "dnf clean all" in the image build any more, so if you have a just-built image, then "dnf install net-tools tcpdump" shouldn't take more than a few seconds to run. Do we really need to preinstall things that are only sometimes needed?

If we are going to keep this, it might be good to split it out into a separate dnf invocation. (One for the things that are actually required to run openshift, and a separate one for extras.)

Reply:
+1
It might be better to make this optional

Reply:
Installing these utils are not taking more than few secs but on a 2 node, 1 master cluster, I had to do it manually on all of them. At least when testing segregating pod traffic, the steps were: launch dind cluster, install debug tools on all nodes/master, run test, got issues, fix code and then repeat. I thought having debug tools by default optimizes developer time over image size overhead. 
Anyway, I will make them optional.